### ISSUE TITLE
Support ARIA

### ISSUE THREAD

**Issue by Th3Whit3Wolf** [maciejhirsz/kobold#76]
It doesn't seem possible right now to add [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) properties or states (i.e. `aria-hidden="true"` or `aria-disabled="true"`).

**Comment by maciejhirsz** [comment]
Good catch! `aria-hidden` is not a Rust identifier since it has a `-` in it, so the parser is failing to consider it. I already have a parse schema for CSS class names, so it should be fairly straight-forward to add it for attributes.
